tests/transformations/subtests/transpile-js/esbuild/index.md

```diff
@@ -0,0 +1,5 @@
+---
+result: partial
+---
+
+esbuild supports tranpiling javascript and typescript code, however, it lacks few transforms when targetting ES5 build target. The output of esbuild can certainly be passed to babel, but esbuild does not provide any mechanism to achieve that.
```

Comment:
```suggestion
esbuild supports transpiling JavaScript and TypeScript code. However, it lacks a few transforms when targeting ES5 build targets. The output of esbuild can be passed to babel, but esbuild does not provide any mechanism to achieve that.
```

Might need to look a bit more closely to decide if there's a way to make this a pass or not. If not, we should file an issue with esbuild and include it here.

Reply:
https://github.com/evanw/esbuild/issues/297

Reply:
From a recent comment on that thread, it looks like https://esbuild.github.io/api/#write could be used here to post-process the output. I'm not 100% sure that makes it a 'pass', but it'd be good to have the example.

This could also be done as a loader plugin. A loader plugin would allow babel to import its polyfills, which could be code-split. The loader route is probably best.

Reply:
Technically the [rules](https://github.com/GoogleChromeLabs/tooling.report/blob/dev/ABOUT.md#rules) do allow custom plugins regardless of their adoption or popularity, so it would be fair to use [a babel plugin](https://github.com/nativew/esbuild-plugin-babel), and change the result to 'pass' here.